.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Good consideration here. I like to see which builds are going to fail, not just that one-and-possibly-more builds failed.

Reply:
👍 . This also mirrors what stuff looks like on Travis.